### DISCUSSION TITLE
2.0-M2 mcl启动报错，但不影响正常启动

### DISCUSSION THREAD

**Discussion by Nambers** [mamoe/mirai#784] [Q&A] [answered]
日志:
20:42:12 [INFO] Verifying "net.mamoe:mirai-console" version 2.0-M2
 20:42:15 [INFO] Verifying "net.mamoe:mirai-console-terminal" version 2.0-M2
 20:42:16 [INFO] Verifying "net.mamoe:mirai-core-all" version 2.0-M2
2020-12-28 20:40:12 I/main: Starting mirai-console...
2020-12-28 20:40:12 I/main: Backend: version 2.0-M2, built on 2020-12-28 18:39:29.
2020-12-28 20:40:12 I/main: Frontend Terminal: version 2.0-M2, provided by Mamoe Technologies
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.

用的是openjdk15

**Comment by Karlatemp** [comment, marked as answer]
某个依赖库使用了slf4j但没有安装对应运行时，仅此而已，对程序运行没啥影响

**Reply by Nambers** [reply]
okok
只是问问，确实没什么影响

**Reply by Karlatemp** [reply]
如果需要显示slf4j打印的日志，添加 https://github.com/project-mirai/mirai-slf4j-bridge
